INTERNALS.md

```diff
@@ -211,3 +201,8 @@ All timestamps are represented as integers, milliseconds since the Unix epoch.
 
   Job identifiers are formatted as batch UIDs.
 
+`counters` (hash)
+
+  Telemetry counters.
```

Comment:
Ah, having read more I see why you've done it; it's to relate it to the underlying statsd metric type. Ignore me.